lib/lita/adapters/slack.rb

```diff
@@ -38,9 +38,28 @@ def roster(target)
         room_roster target.id, api
       end
 
-      def send_messages(target, strings)
+      # @param [Array] messages list of String messages or Symbol emoji reactions
+      # messages starting with the ellipsis character will start a new thread
+      def send_messages(target, messages)
         api = API.new(config)
-        api.send_messages(channel_for(target), strings)
+        channel = channel_for(target)
+        timestamp = try_get(target, :timestamp)
+        thread_ts = try_get(target, :thread_ts)
+
+        strings = messages.select { |s| s.is_a?(String) }
```

Comment:
I don't think we should drop messages if they're not strings.
Edit: oh I see it's the emoji symbol thing?

Reply:
Yeah it was the emoji symbol thing, I think I should drop this logic for filtering and just work on the messages array directly